css/art-feature.css

```diff
@@ -67,8 +73,120 @@ input {
   cursor: pointer;
   font-weight: bold;
   border: none;
-  margin-top: 15px;
+  margin-top: 35px;
+  border-radius: 5px;
+  padding: 15px;
+  font-size: 1rem;
+}
+
+.title-container {
+  display: flex;
+  flex-direction: column;
+}
+
+#instruction-button {
+  position: relative;
+  background-color: #db0979;
+  color: white;
+  cursor: pointer;
+  font-weight: bold;
+  border: none;
+  height: 50%;
+  width: 110px;
+  margin-top: 20px;
   border-radius: 5px;
   padding: 15px;
   font-size: 1rem;
+  transition-duration: 0.4s;
+}
+
+#instruction-button:hover {
+  color: white;
+  box-shadow: 0 12px 16px 0 rgba(0, 0, 0, 0.24),
+    0 17px 50px 0 rgba(0, 0, 0, 0.19);
+}
+
+.instruction-modal {
+  display: none;
+  position: absolute;
+  padding-left: 1rem;
```

Comment:
Can you please use `px` here for consistency ?

Reply:
Made this change